packages/core/src/framework/plugins/augury-plugin-controller.class.ts

```diff
@@ -0,0 +1,57 @@
+import { Subscription } from '../event-emitters';
+import { AuguryPluginWindow } from './augury-plugin-window.interface';
+import { AuguryBridge, AuguryBridgeMessage, AuguryBridgeRequest } from './bridge';
+
+export class AuguryPluginController {
+  private readonly window: AuguryPluginWindow;
```

Comment:
Is the intention that plugins always have windows? Ideally plugins (consumers) could be headless as well as GUI base (or even different intra process).

Reply:
There was no intention really. The existing code was using a window and I just refactored it to separate concerns a bit.